src/UI/Factory.php

```diff
@@ -336,4 +337,62 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers can be used if space is scarce to display secondary information such as a preview of
+	 *     some item or to offer limited user interaction.
+	 *   composition: >
+	 *     Popovers consist of a layer displayed above all other content.
+	 *     The content of the Popover depends on the functionality it performs.
+	 *     A Popover MAY display a title above its content.
+	 *     All Popovers contain a pointer pointing from the Popover to the Triggerer of the Popover.
```

Comment:
Please leave `Triggerer` as this term is also used in the README of the UI Service:
https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/src/UI#abstraction-of-javascript-in-the-framework